tests/Fixer/ClassNotation/VisibilityRequiredFixerTest.php

```diff
@@ -825,6 +825,16 @@ class Foo
                 }
                 PHP,
         ];
+
+        yield 'abstract property hooks' => [
+            <<<'PHP'
+                <?php
+                abstract class Foo
+                {
+                    abstract protected string $bar { get => "a"; set; }
+                }
+                PHP,
```

Comment:
```suggestion

            <<<'PHP'
                <?php
                abstract class Foo
                {
                    abstract public string $a { get => "a"; set; }
                    abstract public string $b { get => "b"; set; }
                    abstract protected string $c { get => "c"; set; }
                }
                PHP,
            <<<'PHP'
                <?php
                abstract class Foo
                {
                    abstract string $a { get => "a"; set; }
                    public abstract string $b { get => "b"; set; }
                    protected abstract string $c { get => "c"; set; }
                }
                PHP,
```

Reply:
tbh adding `public` or reordering modifiers is already covered for properties in other test cases.

Reply:
Exactly the same could be said about not touching `abstract protected`.

It won't hurt having this test case precise to avoid any regression, right?